packages/trader/src/sass/app/modules/reports.scss

```
@@ -24,6 +27,11 @@ $side-padding: 1.2em;
                 max-width: 36rem;
                 margin-left: auto;
             }
+            &-statement {
```

As per suggestion before,

```suggestion
            &--statement {
```